packages/context/src/inject.ts

```diff
@@ -698,3 +700,55 @@ export function describeInjectedProperties(
     ) ?? {};
   return metadata;
 }
+
+/**
+ * Inspect injections for `toClass` and `toProvider`
+ */
+export function inspectInjections(binding: Readonly<Binding<unknown>>) {
+  const json: JSONObject = {};
+  const ctor = binding.valueConstructor ?? binding.providerConstructor;
+  if (ctor == null) return json;
+  const constructorInjections = describeInjectedArguments(ctor, '').map(
+    injectionToJSON,
+  );
+  const propertyInjections = describeInjectedProperties(ctor.prototype);
+  const properties: JSONObject = {};
+  for (const p in propertyInjections) {
+    properties[p] = injectionToJSON(propertyInjections[p]);
+  }
+  if (constructorInjections.length) {
```

Comment:
move  `if (constructorInjections.length) {` block under `const constructorInjections ` to keep them together.